aws/resource_aws_organizations_organization_test.go

```diff
@@ -354,3 +354,91 @@ func testFlattenOrganizationsRootPolicyTypes(t *testing.T, index int, result []m
 		}
 	}
 }
+
+func testAccAwsOrganizationsOrganizationRecreated(t *testing.T, before, after *organizations.Organization) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		if *before.Id == *after.Id {
+			t.Fatalf("Expected change to Org ID value, but both were %v", *before.Id)
+		}
+		return nil
+	}
+}
+
+func testAccAwsOrganizationsOrganization_FeatureSetForcesNew(t *testing.T) {
+	var beforeValue, afterValue organizations.Organization
+	resourceName := "aws_organizations_organization.test"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t); testAccOrganizationsAccountPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAwsOrganizationsOrganizationDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAwsOrganizationsOrganizationConfigFeatureSet(organizations.OrganizationFeatureSetAll),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAwsOrganizationsOrganizationExists(resourceName, &beforeValue),
+					resource.TestCheckResourceAttr(resourceName, "feature_set", organizations.OrganizationFeatureSetAll),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+			{
+				Config: testAccAwsOrganizationsOrganizationConfigFeatureSet(organizations.OrganizationFeatureSetConsolidatedBilling),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAwsOrganizationsOrganizationExists(resourceName, &afterValue),
+					resource.TestCheckResourceAttr(resourceName, "feature_set", organizations.OrganizationFeatureSetConsolidatedBilling),
+					testAccAwsOrganizationsOrganizationRecreated(t, &beforeValue, &afterValue),
+				),
+			},
+		},
+	})
+}
+
+func testAccAwsOrganizationsOrganizationNotRecreated(t *testing.T, before, after *organizations.Organization) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		if *before.Id != *after.Id {
+			t.Fatal("Did not expect change to Org ID value")
+		}
```

Comment:
similar nit as above:

```suggestion
		if aws.StringValue(before.Id) != aws.StringValue(after.Id) {
			return fmt.Errorf("Organization (%s) recreated", aws.StringValue(before.Id))
		}
```